kube/kd/refreshDcuAggregatedCasesView.yaml

```
@@ -28,8 +28,5 @@ spec:
                       key: plaintext
               command: ["/bin/sh", "-c"]
               args:
-                - >
-                  http_status=$( curl -vk -X POST -u ${HOCS_BASICAUTH} -w '%{http_code}' -H 'User-Agent: Refresh DCU_AGGREGATED_CASES'
-                  https://hocs-extracts.{{.KUBE_NAMESPACE}}.svc.cluster.local/admin/export/custom/DCU_AGGREGATED_CASES/refresh );
-                  if [[ $http_status -eq 200 ]]; then exit 0; else exit 1; fi
+                - http_status=$( wget --no-check-certificate --spider -S "https://${HOCS_BASICAUTH}@hocs-audit.{{.KUBE_NAMESPACE}}.svc.cluster.local/admin/export/custom/DCU_AGGREGATED_CASES/refresh" 2>&1 | grep "HTTP/" | awk '{print $2}' ); if [[ $http_status -eq 200 ]]; then exit 0; else exit 1; fi
```

```suggestion
                - http_status=$( wget --no-check-certificate --spider -S "https://${HOCS_BASICAUTH}@hocs-extracts.{{.KUBE_NAMESPACE}}.svc.cluster.local/admin/export/custom/DCU_AGGREGATED_CASES/refresh" 2>&1 | grep "HTTP/" | awk '{print $2}' ); if [[ $http_status -eq 200 ]]; then exit 0; else exit 1; fi
```

~~Is it `extracts` or `extract`?~~ I could just read the line above...